scripts/management/slurm_query_status.py

```diff
@@ -105,12 +106,14 @@ def check_dependancy_met(task, task_list):
 
 
 def get_runnable_tasks(db, n_runs=N_TASKS_TO_RUN, retry_max=RETRY_MAX):
+    do_verification = False
     db_tasks = get_db_tasks_to_be_run(db, retry_max)
     tasks_to_run = []
     for task in db_tasks:
         status = task[2]
         if status == 'created' and check_dependancy_met(task, db_tasks):
-            tasks_to_run.append(task)
+            if task < Process.rrup or do_verification:
```

Comment:
Is it possible for do_verification to be True?

Reply:
Currently not. But will be extended to be an input from somewhere (potentially argparse or calling function)